tests/browserstack_automation/page_objects/topics.page.js

```diff
@@ -0,0 +1,48 @@
+import { $, $$ } from '@wdio/globals';
+import Page from './page';
+
+class TopicsPage extends Page {
+  constructor () {
+    //super().title = 'Terms of Service - We Vote';
+    super().title = 'Pro-choice - WeVote';
+  }
+
+  async load () {
+    await super.open('/value/pro-choice');
+    await super.maximizeWindow();
+    await super.rerender();
+  }
+
+  get getTopicTitleElement() {
+    return $('.IssueName-sc-169wgaf-3 iMcdME')
```

Comment:
I would recommend using id's instead of the class name for the locaters

Reply:
I am not using this element , I will delete this in my next commit.